src/sentry/api/endpoints/event_file_committers.py

```diff
@@ -191,7 +191,7 @@ def get(self, _, project, event_id):
             {commit for match in commit_path_matches for commit in commit_path_matches[match]}
         )
 
-        committers = self._get_committers(annotated_frames, relevant_commits)
+        committers = self._get_committers(project.organization, annotated_frames, relevant_commits)
```

Comment:
nit, if you made this take `organization_id` also, you could just pass `project.organization_id` and potentially save a query (it's likely we've already queried for it somewhere, so not a huge deal)